src/pointer-analysis/goto_program_dereference.h

```diff
@@ -95,10 +95,15 @@ class goto_program_dereferencet:protected dereference_callbackt
 #if 0
   const std::set<irep_idt> *valid_local_variables;
 #endif
-  source_locationt dereference_location;
   goto_programt::const_targett current_target;
 
+  /// Unused
```

Comment:
So the code would compile if we just removed those? Let's just do so in that case.